### PR TITLE
Part 3 of commit towards closing Issue #23 . 

### PR DESCRIPTION
**Pull request by ghost**
Added Albania and Algeria.
Part 3 of commit to close Issue #23

### CODE REVIEW

**Comment by ghost** [comment]
@breck7 : We can review this patch for mistakes sometime ?

File: things/crwb-dz.cdb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+title CRWB
+type registry
+appeared 1995
+standsFor cancer registry of the wilaya of Batna
```
Comment (breck7) [inline things/crwb-dz.cdb:4]: casing on this line.

File: things/tlcr-dz.cdb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+title TLCR
+type registry
+appeared 1999
+standsFor  Tlemcen Cancer Registry
```
Comment (breck7) [inline things/tlcr-dz.cdb:4]: Extra space. I will add trimming to the format command.

File: things/tocr-dz.cdb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+title TOCR
+type registry
+standsFor Tizi-Ouzou Cancer Registry
+description Unknown
+
+reference http://www.dsp-tiziouzou.dz/index.php/ncer
+
```
Comment (breck7) [inline things/tocr-dz.cdb:7]: another reference: https://gco.iarc.fr/today/data/factsheets/populations/12-algeria-fact-sheets.pdf

**Comment by breck7** [comment]
Great stuff @SRS-WRKS !

**Pull request merged**